src/Linux/Native/RunAsUser/util.h

```diff
@@ -0,0 +1,60 @@
+/* ****************************************************************************
```

Comment:
We should figure out a way to share these headers between the C++ projects that we have, so that we don't have multiple copies floating around. Probably a good idea to share libraries like picojson as well.